src/components/SearchResults.js

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from "react";
+
+const SearchResults = ({ bookings }) => {
+  const [selectedCustomerId, setSelectedCustomerId] = useState(null);
+
+  const handleShowProfile = (customerId) => {
+    setSelectedCustomerId(customerId);
+  };
+
+  return (
+    <div>
+      <table className="table table-striped">
+        <thead>
+          <tr>
+            <th>Title</th>
+            <th>First Name</th>
+            <th>Surname</th>
+            <th>Email</th>
+            <th>Room ID</th>
+            <th>Check-in Date</th>
+            <th>Check-out Date</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          {bookings.map((booking) => (
+            <tr key={booking.id}>
```

Comment:
The map works perfectly here and it seems to function as expected. It just seems to be missing lesson 2 task 15. You can achieve it by adding a onClick here and a function to change its class